test/test_autograd.py

```diff
@@ -4459,6 +4459,14 @@ def test_power_function(self):
         c.backward()
         self.assertEqual(b.grad, torch.tensor([-inf, 0., 0.]))
 
+    def test_repeat_zero_dim(self):
```

Comment:
You don't have to do a custom test here. You can simply add a new entry [here](https://github.com/pytorch/pytorch/blob/da4033d32adafd6fe0fa3a3727fcb2b55c19e2e9/torch/testing/_internal/common_methods_invocations.py#L881) that specifies the input size and the arguments.
So something like `('repeat', (2), (0,), 'zero_repeat'),` will do the same thing.